zerver/fixtures/bugdown-data.json

```diff
@@ -121,6 +121,12 @@
       "bugdown_matches_marked": true
     },
     {
+      "name": "escape",
+      "input": "\\**foo**",
+      "expected_output": "<p>**foo**</p>",
+      "bugdown_matches_marked": true
+    },
```

Comment:
I don't understand why this test output would be correct -- what happened to the `\`?

Also, presumably you need to change our echo.js implementation as well?